admin/javascript/src/components/grid-field/row.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import SilverStripeComponent from 'silverstripe-component';
+
+class GridFieldRowComponent extends SilverStripeComponent {
+
+  render() {
+    return (
+      <li className="grid-field-row-component [ list-group-item ]">{this.props.children}</li>
```

Comment:
In fact we're even using display: table etc in the CSS. Table is clearly more semantic than ul.